.github/workflows/reusable_test.yml

```diff
@@ -37,6 +37,9 @@ jobs:
       - name: Install metrics
```

Comment:
Have you considered removing external merger install at all?

Reply:
Shouldn't unlocking version dependency (=install latest tuple-merger and tuple-keydef) solve the issue?

Reply:
Then why do we have the explicit `Remove external tuple merger` steps in some workflow runs?
https://github.com/tarantool/crud/blob/588c215da3699a3c0925fc8b2c0f9aa3640c48c9/.github/workflows/test_on_push.yaml#L105-L107

Reply:
> AFAIC, the external tuple merger is currently outdated (correct me @Totktonada)

It is on track with the built-in merger. There are problems in CI, though.

Reply:
I think that it is good to decouple @CuriousGeorgiy's task from code health effort in `crud`.

I also think that if reducing dependencies in the integration testing workflow helps to run `crud`'s testing in `tarantool`'s testing (and others), it is OK to do.

My thoughts regarding the code health activities are in #365.

Feel free to engage me into a voice discussion if needed.

Reply:
Added a commit that drops external tuple merger and tuple keydef modules installation, as we discussed.